src/styles/work/_programs.scss

```diff
@@ -179,45 +181,57 @@
             line-height: 160%;
             margin: 0 auto;
             max-width: 80vw;
+            @media (max-width: 700px) {
+              font-size: 14px;
+            }
+            @media (max-width: 500px) {
+              font-size: 2.5vw;
+            }
             @media (max-width: 400px) {
               font-size: 11px;
             }
 
           }
         }
-        &.slide-3 {
+        &.slide-map {
```

Comment:
This is indeed more descriptive. And given that all sliders always have that consistency it helps development.